incubator/openfaas/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+name: openfaas
+home: https://blog.alexellis.io/introducing-functions-as-a-service/
```

Comment:
Is the homepage for openfaas really a blog post? Or, is there a more appropriate page?

Reply:
Again I didn't raise this PR @zouyee did, it should be the GitHub page or the homepage www.openfaas.com